index.bs

```diff
@@ -1738,6 +1798,31 @@ the {{CredentialsContainer/create()}} or {{CredentialsContainer/get()}} methods.
     for instance, <code>-7</code> for "ES256" and <code>-257</code> for "RS256".
 </div>
 
+
+### User Verification Requirement enumeration (enum <dfn enum>UserVerificationRequirement</dfn>) ### {#userVerificationRequirement}
+
+<pre class="idl">
+    enum UserVerificationRequirement {
+        "required",
+        "preferred",
+        "discouraged"
+    };
+</pre>
+
+A [=[RP]=] may require [=user verification=] for some of its operations but not for others, and may use this type to express this
+level of requirement.
+
+The value {{UserVerificationRequirement/required}} indicates that the [=[RP]=] requires [=user verification=] for this operation
+and will fail the operation if the response does not have the [=UV=] [=flag=] set.
+
+The value {{UserVerificationRequirement/preferred}} indicates that the [=[RP]=] would like [=user verification=] for this
+operation if possible, but will not fail the operation if the response does not have the [=UV=] [=flag=] set.
+
+The value {{UserVerificationRequirement/discouraged}} indicates that the [=[RP]=] does not require [=user verification=], and does
+not want the [=client=] or [=authenticator=] to request it, but will not fail the operation if the response has the [=UV=]
+[=flag=] set.
+
```

Comment:
> The value {{UserVerificationRequirement/discouraged}} indicates that the [=[RP]=] does not require [=user verification=], and does
not want the [=client=] or [=authenticator=] to request it, 

suggest:
The value {{UserVerificationRequirement/discouraged}} indicates that the [=[RP]=] does not want [=user verification=] employed during the operation (e.g., in the interest of minimizing disruption to the user interaction flow),

Reply:
I still suggest the above update to the language in lines 1821 & 1822.

Reply:
Isn't this addressed in 2969b37 ?

Reply:
yes, thanks.